### ISSUE TITLE
Support running ngrok and the dashboard simultaneously

### ISSUE THREAD

**Issue by agjohnson** [readthedocs/readthedocs.org#10960]
Our local dev environment should allow for ngrok to run in the background and respond to Stripe webhooks while leaving the normal dashboard and dashboard URL untouched.

Instead, what is happening is:

- I start the instance with Stripe test secrets
- I run my instance with `ngrok http 80; inv docker.up --http-domain readthedocs.ngrok.io` as per the dev docs
- Stripe works
- But the dashboard is now not available at `devthedocs.com`
- Trying to load the dashboard (at least the new dashboard) at `readthedocs.ngrok.io`, none of the styles or JS loads correctly

Instead:

- I start the instance with Stripe test secrets from `.envrc`
- Dashboard is accessible at the normal `devthedocs.com`
- Ngrok is started and something translates the webhooks to `devthedocs.com/djstripe/webhook` instead of `readthedocs.ngrok.io`

**Comment by stsewd** [comment]
I've been using https://webhook.site/ for anything that requires a server-to-server connection.

**Comment by humitos** [comment]
Are you updating the webhook.site URL every time you start a new "instance" or that unique URL will work forever? Do you also use the `whcli` CLI tool as well to forward to your local instance?

**Comment by stsewd** [comment]
> Are you updating the webhook.site URL every time you start a new "instance" or that unique URL will work forever?

URLs expire after some days or weeks, but I think you can keep it if you generate the URL while you are signed in. Personally, I prefer for URL to expire after some days.

> Do you also use the whcli CLI tool as well to forward to your local instance?

yep, that's required

**Comment by humitos** [comment]
> URLs expire after some days or weeks, but I think you can keep it if you generate the URL while you are signed in.

It seems that to sign in you have to pay for a plan.

> Personally, I prefer for URL to expire after some days.

Why? Don't you have to update those URL on every service you use when that happens? I mean, going to the GHA and setup the webhook URL... etc..

I'm trying to think if webhook.site could be a replacement for our ngrok setup. If so, we should document it and make it work for the whole team.

**Comment by stsewd** [comment]
> Why? Don't you have to update those URL on every service you use when that happens? I mean, going to the GHA and setup the webhook URL... etc..

yeah, but I prefer doing that work every week or so instead of having a static URL that connects to my computer. It's not like I work every day on server-to-server integrations.

**Comment by humitos** [comment]
I've been using webhook.site with the whcli tool as well and I'm pretty happy with it. I think we could remove everything related to ngrok and stop paying its subscription.

**Comment by agjohnson** [comment]
I tried cancelling but don't have permission on my account, looks like you are the only user that can update the subscription currently @humitos

**Comment by humitos** [comment]
I downgraded the NGROK subscription to Free 👍🏼 

> URLs expire after some days or weeks, but I think you can keep it if you generate the URL while you are signed in.

I found myself wanting to extend the period of a URL more than 7 days. This 7-days period renews every time a new webhook is received. So, it's easy to renew and keep the URL for more than 7 days 👍🏼
